e2e/tests/user/login.spec.ts

```diff
@@ -1,40 +1,25 @@
 import GoogleLoginPage, {selectors} from '../../app/google-login';
-import PuppeteerLaunch from '../../driver/puppeteer-launch';
 
 const configs = require('../../resources/workbench-config');
 
-// set timeout globally per suite, not per test.
-jest.setTimeout(2 * 60 * 1000);
 
 describe('Login tests:', () => {
-  let browser;
-  let incognitoContext;
-  let page;
-
-  beforeAll(async () =>  {
-    browser = await PuppeteerLaunch();
-  });
 
   beforeEach(async () => {
-    incognitoContext = await browser.createIncognitoBrowserContext();
-    page = await incognitoContext.newPage();
     await page.setUserAgent(configs.puppeteerUserAgent);
-    await page.setDefaultNavigationTimeout(60000);
+    await page.setDefaultNavigationTimeout(120000);
```

Comment:
For now, `setUserAgent` and `setDefaultNavigationTimeout` are required in `beforeEach`, still is boilerplate code. I have a PR in work that will remove them from test files for good. I didn't want to pack any more changes in this PR.